libnd4j/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.6)
 project(libnd4j)
-set( CMAKE_VERBOSE_MAKEFILE on)
+set(CMAKE_VERBOSE_MAKEFILE OFF)
```

Comment:
Please ensure @sshepel ok with this change, they've used verbose mode to avoid jenkins timeouts